docs/source/ipython_projects.rst

```diff
@@ -1,4 +1,4 @@
-IPython Projects
+IPython projects
```

Comment:
I don't think this needs to be a separate page, let's list them in the categorical index. IPython is a project of the Jupyter team.